### ISSUE TITLE
Be able to add to firewall list, programs that are installed on another user profiles, but not in the main one.

### ISSUE THREAD

**Issue by tv2q0** [ukanth/afwall#1358]
Hi, i´m testing Afwall kind recently, essentially because i need a firewall that lets use simultaneously a VPN; but also i have several specific requirements. I own a Tecno Spark 8 T, android 11, magisk rooted, and recently i could unlock its multiple user support, so i decided to create two new user besides of the main one (proprietary); so i know this firewall has multi user support, but really haven't could make it work fine on these user profiles, but that's for another topic. The feature that i want for it, is that i could select for blocking/allow an app that is installed on any of the other user prifles, but is not in the main one (where is currently installed Afwall). Currently, only appears on the list, apps that are installed just on the main user profile, but if any app is installed on another profile but not in the main one, it doesn't appear.
Besides, it would be wonderful if Afwall will work on non rooted   user profiles. I know thatg magisk also have support to give root access to other user profiles, inside the rooted phone; but one of the profile that i created, is for banking apps, so you know that banking apps and root access don't get along fine, so i would be nice to can work that user profile, without having to get root access.

**Comment by ghost** [comment]
Yes this would be great! Took me a while to figure out why some work profile apps appeared in AFWall+ and others didn't, only works apps that are also in the main profile appear. 

As OP is requesting with this ticket, it would be great to not require a work app to be installed in the main profile for it to appear in AFWall+
